src/Microsoft.TestPlatform.CrossPlatEngine/Utilities/DiscoveredSourceHelper.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+using System.Linq;
+
+using Microsoft.VisualStudio.TestPlatform.ObjectModel.Client;
+
+namespace Microsoft.VisualStudio.TestPlatform.CrossPlatEngine.Utilities;
+
+internal class DiscoveredSourceHelper
```

Comment:
Class could be made `static`.

```suggestion
internal static class DiscoveredSourceHelper
```